### ISSUE TITLE
IE 에서 최초 로그인시, 접속 오류

### ISSUE THREAD

**Issue by LIMJK** [xpressengine/xe-core#2320]
[IE 11.55.17763.0] 브라우저를 열어 위젯 로그인창을 열어 아이디와 비밀번호 입력하고 로그인하려면,
![default](https://user-images.githubusercontent.com/24222320/46861595-f26f8a80-ce4d-11e8-9ddb-c220f7d540a9.PNG)
와 같은 오류창이 열립니다. 여기서 그냥 [로그인]을 누르면, 화면이 바뀌면서 ~
![01](https://user-images.githubusercontent.com/24222320/46861753-5abe6c00-ce4e-11e8-8934-84598217fcee.PNG)
이 열립니다. 그래서 이곳에 아이디와 비밀번호를 입력하고 로그인하면,
![02](https://user-images.githubusercontent.com/24222320/46861775-6ca00f00-ce4e-11e8-8896-09f1ed93ea4f.PNG)
이 열립니다. 그래서 다시 위젯 로그인이나 이곳에 아이디와 비밀번호를 재입력하면 정상적으로 로그인됩니다. 이는 [IE] 브라우저를 처음 열었을 때, 해당되고~ 열어 놓은 상태에서 로그아웃했다가 로그인하면 정상적으로 로그인이 됩니다.

이외에 크롬 69.0.3497.100, 네이버 웨일 최신판 사용중이나, 이 두가지 모두 오류 없이 잘 로그인됩니다.
유독 IE만 문제네요.
참고로 3번째 오류인, [기본 URL 설정이 안되어 있습니다.] 부분은 관리자창의 [설정]-[일반]-[고급]-[기본 URL]을 뜻하는 것 같은데요. 이곳에는 해당 도메인명이 정상적으로 입력되어 있습니다.

위에서 2번째 오류 메시지가 나왔을 때,
열린 로그인 입력창이 아닌 위젯 로그인창을 다시 열어 아이디와 비밀번호를 입력하면,
오류 없이 바로 로그인됩니다.

저만 그런 걸까요? 아니면 오류일까요???

**Comment by dorami** [comment]
/files/config/db.config.php 에서 $db_info->default_url 부분이 정상적으로 있는지 확인해 보세요.

**Comment by LIMJK** [comment]
> /files/config/db.config.php 에서 $db_info->default_url 부분이 정상적으로 있는지 확인해 보세요.

네, 위 질문에 작성했듯이 /files/config/db.config.php 파일의 $db_info->default_url 위치에는 운영 도메인명이 그대로 있습니다.

**Comment by kms0219kms** [comment]
본 XE1 버전의 개발은 작년 2월부터 사실상 중단 상태입니다.

이 문제가 계속된다면, XE1의 업데이트 버전인 [Rhymix](https://rhymix.org)으로 업데이트 해보시기 바랍니다.

업데이트 방법은 [여기](https://rhymix.org/manual/introduction/xe-upgrade)를 참고하시면 됩니다.

제대로 된 답변 드리지 못한 점 죄송합니다.
